code/__HELPERS/unsorted.dm

```diff
@@ -1496,6 +1496,7 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 		/obj/item/reagent_containers/food/snacks/clothing
 		)
 	blocked |= typesof(/obj/item/reagent_containers/food/snacks/customizable)
+	blocked |= /obj/item/reagent_containers/food/snacks
```

Comment:
Why is this an |= rather than just putting it in the list above?

Reply:
Figured I'd preserve the format established earlier, and keep the typesof list on its own line. I can change it if you think that's unnecessary.

Reply:
isn't the only reason the line above is separate because it's a list intersection, not a single item?

Reply:
Oh, I'm silly, I thought it was done on its own like in `get_random_drink()`. The "format established earlier" was not actually a format that was established!

I'll fix this when I get back from work.